app/forms/nsm/steps/case_disposal_form.rb

```diff
@@ -5,6 +5,7 @@ class CaseDisposalForm < ::Steps::BaseFormObject
       attribute :plea_category
       validates :plea, presence: true, inclusion: { in: PleaOptions.values }
 
+      # rubocop:disable Style/HashEachMethods
```

Comment:
Not going to put this for every single one, but if we can instead fix the issue that would be preferable

Reply:
hmm, not sure this is something that should be addressed and if it should I feel like it would need to be done as a separate ticket to think about how we deal with value objects. The ValueObject is not a hash and therefore we don't have access to the each_value method - so not sure why it's calling out that we are not using hash methods.

Reply:
As I say it was a general remark that if we can resolve the issues we should, but feel free to write another ticket for it.

Reply:
Yeah I see your point - I generally went down the route of: if there is value in conforming to the linting rule, fix it, if not - leave the ignore in. In this case it was that it seems to be applying a hash related style cop to a non hash object (though maybe there's something I'm not seeing there). Other examples is Rspec/ExampleLength for tests on things like tables as those naturally will just be long (couldn't think of a way to shorten them that didn't require unnecessary complexity or unnecessary splitting up of tests and therefore adding to test execution time)

Will have another pass, but not looking to add another ticket if we can't find tangible value from it